sale_automatic_workflow_stock_location_route/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,2 @@
+This allow to automatically set a specific route on sale lines with product delivered by stock move, when the sale is confirmed.
+Just set the desired route on the sale automatic workflow.
```

Comment:
Maybe add a small comment to indicates the configured route will erase any other route set manually?